src/model/wallet.js

```diff
@@ -360,24 +380,33 @@ export const createWallet = async (
         (type === WalletTypes.readOnly || isPrivateKeyOverwritingSeedMnemonic)
       ) {
         Alert.alert('Oops!', 'Looks like you already imported this wallet!');
+        logger.sentry('[createWallet] - already imported this wallet');
         return null;
       }
     }
 
     const id = existingWalletId || `wallet_${Date.now()}`;
+    logger.sentry('[createWallet] - wallet ID', { id });
+
+    // Save seed - save this first
+    await saveSeedPhrase(walletSeed, id);
+    logger.sentry('[createWallet] - saved seed phrase');
 
     // Save address
     await saveAddress(wallet.address);
+    logger.sentry('[createWallet] - saved address');
+
     // Save private key
     await savePrivateKey(wallet.address, wallet.privateKey);
-    // Save seed
-    await saveSeedPhrase(walletSeed, id);
+    logger.sentry('[createWallet] - saved private key');
```

Comment:
We should add the same logs while saving the PKEY of the accounts from autodiscovery